src/webhook/test_namespace_validation.py

```diff
@@ -67,6 +21,51 @@ class FakeRequest(object):
 
 
 class TestNamespaceValidation(unittest.TestCase):
+    PRIVILEGED_NAMESPACES = (
+        "kube-admin",
+        "kube-foo",
+        "kube-",
+        "openshift",
+        "openshifter",
+        "openshift-foo",
+        "ops-health-monitoring",
+        "management-infra",
+        "default",
+        "logging",
+        "sre-app-check",
+        "redhat-user",
+        "redhat-",
```

Comment:
To be in line with changes in https://github.com/openshift/managed-cluster-validating-webhooks/pull/66, I'll make the same suggested changes here.

`redhat-` is not a valid Namespace name, and so it doesn't make sense to test it. The Kubernetes API would deny the creation of such a Namespace.
```suggestion
```

Reply:
done